snowflake/1.1/modules/ROOT/pages/snowflake-connector-examples.adoc

```diff
@@ -0,0 +1,122 @@
+= Snowflake Connector 1.1 Examples - Mule 4
+
+The following example returns all rows of a table based on a watermark.
+
+These examples use variables for some field values. You can either:
+
+* Replace the variables with their values in the code.
+* Provide the values for each variable in a properties file and then refer to that file from the connector configuration.
+
+If you don’t know how to use a properties file, refer to xref:mule-runtime::mule-app-properties-to-configure.adoc[Configuring Property Placeholders].
+
+Before you try the examples, access Anypoint Studio (Studio) and verify that the *Mule Palette* view displays entries for Snowflake Connector. If not, follow the instructions in xref:snowflake-connector-studio.adoc#add-connector-to-project[Add the Connector to Your Mule Project].
+
+== Return All Rows of a Table
+
+This example shows how to use the *On Table Row* source to return rows of a table based on a watermark.
```

Comment:
return rows that have the watermark column greater than the last watermark processed